courses/free-courses-en.md

```diff
@@ -31,6 +31,7 @@
     * [Ansible](#ansible)
     * [Chef](#chef)
 * [Digital Electronics](#digital-electronics)
+* [Digital Image processing](#digital-image-processing)
```

Comment:
It's usually not helpful to add a category for a single resource. Try to find another category, or see if there are other books in our lists that would fit into your category.

Use MATLAB or Misc categories, or start a "Graphics Programming" category, moving the "Computer Grephics" couse from Misc.